src/main/java/com/spotinst/sdkjava/model/bl/admin/account/Account.java

```diff
@@ -0,0 +1,114 @@
+package com.spotinst.sdkjava.model.bl.admin.account;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Account {
+    //region Members
+    @JsonIgnore
+    private Set<String>                    isSet;
+    private String                         id;
+    private String                         name;
+    private String                         organizationId;
```

Comment:
Please check the organizationId datatype

Reply:
It is string as per postman collection.